connman/plugins/sailfish_ofono.c

```diff
@@ -671,7 +672,7 @@ static gboolean modem_delayed_connect(gpointer data) {
 	return G_SOURCE_REMOVE;
 }
 
-static void clean_modem_delayed_connect(struct modem_data *md)
+static void modem_clear_delayed_connect(struct modem_data *md)
```

Comment:
If not too much trouble, could you move these renames to the commit that introduces this function, would be less noise that way.

Reply:
Done.